app.fotema.Fotema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "app.fotema.Fotema",
```

Comment:
I don't see an webpage at fotema.app Is the domain yours? How can I verify?

Reply:
I bought the domain, but have only setup email. Feel free to email hello@fotema.app and I will reply.

Reply:
Preferably, just setting up a blank page and linking https://github.com/blissd/fotema will be enough.

If you can't do that please send an email to `admins@flathub.org`, confirming the submission.

Reply:
https://fotema.app/ now exists and points to the Fotema github repo.

Reply:
You have a HSTS header set I can't open the page

Reply:
![Screenshot_20240512_184905_Firefox Nightly.jpg](https://github.com/flathub/flathub/assets/62639087/e43301b3-8bc9-4304-8950-fdc263d722e2)

Reply:
Are you using http instead of https?

Reply:
I think HSTS is mandatory for .app domains.

Reply:
I can see the full webpage now, maybe the changes you were making were still propagating. 

Thanks.